<?xml version="1.0" encoding="utf-8"?>
<cvrfdoc xmlns="http://www.icasi.org/CVRF/schema/cvrf/1.1" xmlns:cvrf="http://www.icasi.org/CVRF/schema/cvrf/1.1">
  <DocumentTitle xml:lang="en">Red Hat Security Advisory: Red Hat OpenStack Platform 16.2 (python-django20) security update</DocumentTitle>
  <DocumentType>Security Advisory</DocumentType>
  <DocumentPublisher Type="Vendor">
    <ContactDetails>secalert@redhat.com</ContactDetails>
    <IssuingAuthority>Red Hat Product Security</IssuingAuthority>
  </DocumentPublisher>
  <DocumentTracking>
    <Identification><ID>RHSA-2021:3490</ID></Identification>
    <Status>Final</Status>
    <Version>1</Version>
    <RevisionHistory>
       <Revision>
         <Number>1</Number>
         <Date>2021-09-15T13:41:00Z</Date>
         <Description>Current version</Description>
       </Revision>
    </RevisionHistory>
    <InitialReleaseDate>2021-09-15T13:41:00Z</InitialReleaseDate>
    <CurrentReleaseDate>2021-09-15T13:41:00Z</CurrentReleaseDate>
    <Generator>
      <Engine>Red Hat SDEngine 3.0.2</Engine>
      <Date>2021-09-15T18:02:00Z</Date>
    </Generator>
  </DocumentTracking>
  <DocumentNotes>
    <Note Title="Topic" Type="Summary" Ordinal="1" xml:lang="en">
An update for python-django20 is now available for Red Hat OpenStack
Platform 16.2 (Train).

Red Hat Product Security has rated this update as having a security impact
of Moderate. A Common Vulnerability Scoring System (CVSS) base score, which
gives a detailed severity rating, is available for each vulnerability from
the CVE link(s) in the References section.
    </Note>
    <Note Title="Details" Type="General" Ordinal="2" xml:lang="en">
Security Fix(es):

* Potential directory-traversal via archive.extract() (CVE-2021-3281)

* Potential directory traversal via ``admindocs`` (CVE-2021-33203)

* Possible indeterminate SSRF RFI and LFI attacks since validators accepted
leading zeros in IPv4 addresses (CVE-2021-33571)

For more details about the security issue(s), including the impact, a CVSS
score, acknowledgments, and other related information, refer to the CVE
page listed in the References section.
    </Note>
    <Note Title="Terms of Use" Ordinal="3" Type="Legal Disclaimer" xml:lang="en">This content is licensed under the Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). If you distribute this content, or a modified version of it, you must provide attribution to Red Hat Inc. and provide a link to the original.</Note>
  </DocumentNotes>
  <DocumentDistribution xml:lang="en">Copyright Â© 2021 Red Hat, Inc. All rights reserved.</DocumentDistribution>
  <AggregateSeverity Namespace="https://access.redhat.com/security/updates/classification/">Moderate</AggregateSeverity>
  <DocumentReferences>
    <Reference Type="Self">
       <URL>https://access.redhat.com/errata/RHSA-2021:3490</URL>
       <Description>https://access.redhat.com/errata/RHSA-2021:3490</Description>
    </Reference>
    <Reference Type="External">
       <URL>https://access.redhat.com/security/updates/classification/#moderate</URL>
       <Description>https://access.redhat.com/security/updates/classification/#moderate</Description>
    </Reference>
  </DocumentReferences>
  <ProductTree xmlns="http://www.icasi.org/CVRF/schema/prod/1.1">
    <Branch Type="Product Family" Name="Red Hat OpenStack Platform">
      <Branch Type="Product Name" Name="Red Hat OpenStack Platform 16.2">
        <FullProductName ProductID="8Base-RHOS-16.2" CPE="cpe:/a:redhat:openstack:16.2::el8">Red Hat OpenStack Platform 16.2</FullProductName>
      </Branch>
    </Branch>
    <Branch Type="Product Version" Name="python-django20-0:2.0.13-16.el8ost.1">
      <FullProductName ProductID="python-django20-0:2.0.13-16.el8ost.1" CPE="cpe:/a:redhat:openstack:16.2::el8">python-django20-2.0.13-16.el8ost.1.src.rpm</FullProductName>
    </Branch>
    <Relationship ProductReference="python-django20-0:2.0.13-16.el8ost.1" RelationType="Default Component Of" RelatesToProductReference="8Base-RHOS-16.2">
      <FullProductName ProductID="8Base-RHOS-16.2:python-django20-0:2.0.13-16.el8ost.1" CPE="cpe:/a:redhat:openstack:16.2::el8">python-django20-0:2.0.13-16.el8ost.1 as a component of Red Hat OpenStack Platform 16.2</FullProductName>
    </Relationship>
  </ProductTree>

  <Vulnerability Ordinal="1" xmlns="http://www.icasi.org/CVRF/schema/vuln/1.1">
    <Notes><Note Title="Vulnerability Description" Type="General" Ordinal="1" xml:lang="en">django: Potential directory-traversal via archive.extract()</Note></Notes>
    <DiscoveryDate>2021-01-25T00:00:00Z</DiscoveryDate>
    <ReleaseDate>2021-02-01T10:00:00Z</ReleaseDate>
    <Involvements><Involvement Party="Vendor" Status="Completed"></Involvement></Involvements>
    <CVE>CVE-2021-3281</CVE>
    <ProductStatuses><Status Type="Fixed">
      <ProductID>8Base-RHOS-16.2:python-django20-0:2.0.13-16.el8ost.1</ProductID>
    </Status></ProductStatuses>
    <Threats><Threat Type="Impact"><Description>Moderate</Description></Threat></Threats>
    <Remediations>
      <Remediation Type="Vendor Fix"><Description xml:lang="en">
For details on how to apply this update, which includes the changes
described in this advisory, refer to:

https://access.redhat.com/articles/11258
    </Description>
      <URL>https://access.redhat.com/errata/RHSA-2021:3490</URL></Remediation>
    </Remediations>
    <References>
      <Reference>
        <URL>https://access.redhat.com/security/cve/CVE-2021-3281</URL>
        <Description>CVE-2021-3281</Description>
      </Reference>
      <Reference>
        <URL>https://bugzilla.redhat.com/show_bug.cgi?id=1919969</URL>
        <Description>bz#1919969: CVE-2021-3281 django: Potential directory-traversal via archive.extract()</Description>
      </Reference>
    </References>
  </Vulnerability>

  <Vulnerability Ordinal="2" xmlns="http://www.icasi.org/CVRF/schema/vuln/1.1">
    <Notes><Note Title="Vulnerability Description" Type="General" Ordinal="1" xml:lang="en">django: Potential directory traversal via ``admindocs``</Note></Notes>
    <DiscoveryDate>2021-05-26T00:00:00Z</DiscoveryDate>
    <ReleaseDate>2021-06-02T00:00:00Z</ReleaseDate>
    <Involvements><Involvement Party="Vendor" Status="Completed"></Involvement></Involvements>
    <CVE>CVE-2021-33203</CVE>
    <ProductStatuses><Status Type="Fixed">
      <ProductID>8Base-RHOS-16.2:python-django20-0:2.0.13-16.el8ost.1</ProductID>
    </Status></ProductStatuses>
    <Threats><Threat Type="Impact"><Description>Moderate</Description></Threat></Threats>
    <Remediations>
      <Remediation Type="Vendor Fix"><Description xml:lang="en">
For details on how to apply this update, which includes the changes
described in this advisory, refer to:

https://access.redhat.com/articles/11258
    </Description>
      <URL>https://access.redhat.com/errata/RHSA-2021:3490</URL></Remediation>
    </Remediations>
    <References>
      <Reference>
        <URL>https://www.djangoproject.com/weblog/2021/jun/02/security-releases/</URL>
        <Description>https://www.djangoproject.com/weblog/2021/jun/02/security-releases/</Description>
      </Reference>
      <Reference>
        <URL>https://access.redhat.com/security/cve/CVE-2021-33203</URL>
        <Description>CVE-2021-33203</Description>
      </Reference>
      <Reference>
        <URL>https://bugzilla.redhat.com/show_bug.cgi?id=1966251</URL>
        <Description>bz#1966251: CVE-2021-33203 django: Potential directory traversal via ``admindocs``</Description>
      </Reference>
    </References>
  </Vulnerability>

  <Vulnerability Ordinal="3" xmlns="http://www.icasi.org/CVRF/schema/vuln/1.1">
    <Notes><Note Title="Vulnerability Description" Type="General" Ordinal="1" xml:lang="en">django: Possible indeterminate SSRF, RFI, and LFI attacks since validators accepted leading zeros in IPv4 addresses</Note></Notes>
    <DiscoveryDate>2021-05-26T00:00:00Z</DiscoveryDate>
    <ReleaseDate>2021-06-02T00:00:00Z</ReleaseDate>
    <Involvements><Involvement Party="Vendor" Status="Completed"></Involvement></Involvements>
    <CVE>CVE-2021-33571</CVE>
    <ProductStatuses><Status Type="Fixed">
      <ProductID>8Base-RHOS-16.2:python-django20-0:2.0.13-16.el8ost.1</ProductID>
    </Status></ProductStatuses>
    <Threats><Threat Type="Impact"><Description>Moderate</Description></Threat></Threats>
    <Remediations>
      <Remediation Type="Vendor Fix"><Description xml:lang="en">
For details on how to apply this update, which includes the changes
described in this advisory, refer to:

https://access.redhat.com/articles/11258
    </Description>
      <URL>https://access.redhat.com/errata/RHSA-2021:3490</URL></Remediation>
    </Remediations>
    <References>
      <Reference>
        <URL>https://www.djangoproject.com/weblog/2021/jun/02/security-releases/</URL>
        <Description>https://www.djangoproject.com/weblog/2021/jun/02/security-releases/</Description>
      </Reference>
      <Reference>
        <URL>https://access.redhat.com/security/cve/CVE-2021-33571</URL>
        <Description>CVE-2021-33571</Description>
      </Reference>
      <Reference>
        <URL>https://bugzilla.redhat.com/show_bug.cgi?id=1966253</URL>
        <Description>bz#1966253: CVE-2021-33571 django: Possible indeterminate SSRF, RFI, and LFI attacks since validators accepted leading zeros in IPv4 addresses</Description>
      </Reference>
    </References>
  </Vulnerability>
</cvrfdoc>
